storage/persistent_storage/rosa-persistent-storage-aws-ebs.adoc

```diff
@@ -0,0 +1,70 @@
+====
+
+The Kubernetes persistent volume framework enables the administrators to provision a cluster with persistent storage and gives users a way to request those resources without having any knowledge of the underlying infrastructure. You can dynamically provision the AWS EBS volumes. Persistent volumes are not bound to a single project or namespace; therefore, the volumes can be shared across ROSA clusters. Persistent volume claims are specific to a project or namespace and can be requested by users.
+
+[IMPORTANT]
+====
+ROSA defaults to using an in-tree or non-Container Storage Interface (CSI) plug-in to provision AWS EBS storage. In future ROSA versions, volumes provisioned using existing in-tree plug-ins are planned for migration to their equivalent CSI driver. After full migration, the in-tree plug-ins will eventually be removed from the future versions of ROSA.
+====
+
+[IMPORTANT]
+====
+High-availability of storage in the infrastructure is left to the underlying storage provider.
+====
+
+== Format of persistent volumes
+Before a ROSA cluster mounts the volume and passes it to a container, the cluster checks that the volume contains a file system as specified by the *fsType* parameter in the persistent volume definition. If the device is not formatted with the file system, all data from the device is erased and the device is automatically formatted with the given file system.
+This verification enables you to use unformatted AWS volumes as persistent volumes, as the ROSA cluster formats the AWS volumes before the first use.
+
+
+== Capacity of EBS volumes on a node
+By default, a ROSA cluster supports a maximum of 39 EBS volumes attached to one node. This limit is consistent with the link:https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/volume_limits.html#linux-specific-volume-limits[AWS volume limits]. The volume limit depends on the instance type.
+
+[IMPORTANT]
+====
+You must use either in-tree or CSI volumes and their respective storage classes, but never both volume types at the same time. The maximum attached EBS volume number is counted separately for in-tree and CSI volumes.
+====
+
+== Creating a persistent volume claim
+
+.Prerequisites
+Storage must exist in the underlying infrastructure before it can be mounted as a volume in the ROSA cluster.
+
+.Procedure
+. In the OpenShift Cluster Manager (OCM) console, click *Storage → Persistent Volume Claims*.
+. In the persistent volume claims overview, click *Create Persistent Volume Claim*.
+. Define the desired options on the page that appears.
+  .. Select the storage class created previously from the drop-down menu.
+  .. Enter a unique name for the storage claim.
+  .. Select the access mode. This selection determines the read and write access for the newly created storage claim.
+  .. Define the size of the storage claim.
+. Click *Create* to create the persistent volume claim and generate a persistent volume.
+
+== Additional resources
```

Comment:
Check this against https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#additional-resources-sections
Needs some tagging, should also be a list not a paragraph.

Reply:
Thanks, made changes as necessary.